repos/common/.leapp/info

```diff
@@ -1 +1 @@
-{"messages": {}, "name": "common", "id": "efcf9016-f2d1-4609-9329-a298e6587b3c"}
\ No newline at end of file
+{"messages": {}, "name": "common", "id": "efcf9016-f2d1-4609-9329-a298e6587b3c"}
```

Comment:
@ZhukovGreen This should be fixed in leap first as this is the real output from leapp and it should not be different in our repositorz.

Reply:
Added this to ignore